.fixtures.yml

```diff
@@ -2,10 +2,10 @@ fixtures:
   repositories:
     stdlib:
       repo: git://github.com/puppetlabs/puppetlabs-stdlib
-      ref: '4.6.0'
+      ref: '4.20.0'
```

Comment:
As on the other PR: We should unpin those modules.

Reply:
updated